modules/instruments/api/instruments-routes.ts

```diff
@@ -0,0 +1,63 @@
+import { Request, Response } from 'express';
+import * as core from 'express-serve-static-core';
+import instrumentsList from '../data/instruments';
+
+module.exports = function (app: core.Express) {
+    /**
+     * @openapi
+     * '/instruments':
+     *   get:
+     *     tags:
+     *     - Instruments
+     *     summary: Obtain a list of all instruments
+     *     responses:
+     *       '200':
+     *         description: OK
+     *         schema:
+     *           type: array
+     *           items:
+     *             type: string
+     *             example: Saxhorn, Tuba
+     */
+    app.get('/instruments/', (req: Request, res: Response) => {
+        res.json({
```

Comment:
Hi, do you mind adding a qty attribute to the end of this endpoint so a user can specify the amount they would like returned. We have a route util for this as well called getQtyFromRequest

Reply:
Sure :D

Reply:
@ageddesi Should be done with the tests updated :)